src/Clients/CaradhrasDataApiClient.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Idez\Caradhras\Clients;
+
+use Carbon\Carbon;
```

Comment:
We can use the `CarbonInterface` here

```suggestion
use Carbon\CarbonInterface;
```